telosnetwork/teloscan
# #251 | Date formatting

# Fixes #251 

## Description
This PR updates the way dates are displayed sitewide

- updates the way full dates are displayed (switch to clearer month display, switch to 24 hour time, add timezone offset)
- add info icon and tooltip to date fields
- fix transaction table header (broken in previous quasar upgrade, table header was not displaying the correct tooltips)

## Test scenarios
- go to a transaction table, eg. https://deploy-preview-289--teloscan-stage.netlify.app/
    - there should be tooltips on the Date and Method fields of the transaction table
    - there should be no tooltip or info icon next to the actual dates in the table 
- click the info icon next to the Date header
    - dates in the table should all change format
    - new date format should include UTC offset
- go to a transaction page, eg. https://deploy-preview-289--teloscan-stage.netlify.app/tx/0xbda210a6913fd7f85cb7efd68b3f16a7ffdd7cdde7f34936d422309671cc438b
    - date field should match previous page, but should also have an info icon and tooltip on hover
- click the date field
    - date format should change to new format

## Checklist:
